src/tasks/BagelAITools.py

```
@@ -2,21 +2,29 @@
+from ok import TaskDisabledException, og
 from qfluentwidgets import FluentIcon
 
 from src.tasks.BaseNTETask import BaseNTETask
 from src.tasks.NTEOneTimeTask import NTEOneTimeTask
 
+def get_app_locale() -> bool:
+    """get app locale."""
+
+    try:
+        return og.app.locale.name()
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)